src/main/java/net/neoforged/neoforge/registries/datamaps/builtin/NeoForgeDataMaps.java

```diff
@@ -57,5 +72,6 @@ private static ResourceLocation id(final String name) {
     private static void register(final RegisterDataMapTypesEvent event) {
         event.register(COMPOSTABLES);
         event.register(VIBRATION_FREQUENCIES);
+        event.register(PARROT_IMITATIONS);
```

Comment:
Presuming the order of these doesn't matter in regards to backwards compat (which I am assuming it doesn't given they just end up in a `HashMap`), do we want to potentially self impose a style guide of keeping this alphabetical in our code? Rather than just appending them haphazardly to the end? Same potentially with actually declaring the `DataMapType` field order.

Reply:
I would go for a variation of that. Alphabetical but grouping maps for the same registry one after the other.